Riot/Modules/directory/people/detail/PeopleDetailViewController.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  PeopleDetailViewController.swift
+//  Riot
+//
+//  Created by Naurin Afrin on 5/8/20.
+//  Copyright © 2020 matrix.org. All rights reserved.
+//
+
+import UIKit
+
+
+private enum SectionType: Int, CaseIterable {
+    case PEOPLE_PROFILE = 0
+    case PHONE_CELL = 1
+    case EMAIL_CELL = 2
+    case ROLE_CELL = 3
+    
+    var cellIdentifier: String {
+        switch self {
+        case .PEOPLE_PROFILE:
+            return "PeopleProfileTableViewCell"
+        case .PHONE_CELL:
+            return "PhoneTableViewCell"
+        case .EMAIL_CELL:
+            return "EmailTableViewCell"
+        case .ROLE_CELL:
+            return "RoleTableViewCell"
+        }
+    }
+}
+
+class PeopleDetailViewController: UIViewController {
+    @IBOutlet weak var tableView: UITableView!
+    var actPeople: ActPeople?
+    
+    var roles = [
+        RoleModel(name: "Test1", description: "Emergency Department", isExpanded: false),
```

Comment:
Please add a TODO comment that this should be replaced once the back end is available

Reply:
Added TODO message at this point.